bin/arg.ml

```diff
@@ -29,9 +29,17 @@ module Dep = struct
 
   let file s = Dep_conf.File (String_with_vars.make_text Loc.none s)
 
-  let alias s = Dep_conf.Alias (String_with_vars.make_text Loc.none s)
+  let alias_gen make ~dir s =
+    let path =
+      Dune.Alias.Name.to_string s
+      |> Stdune.Path.Local.relative dir
+      |> Stdune.Path.Local.to_string
+    in
+    make (String_with_vars.make_text Loc.none path)
+
+  let alias = alias_gen (fun x -> Dep_conf.Alias x)
 
-  let alias_rec s = Dep_conf.Alias_rec (String_with_vars.make_text Loc.none s)
+  let alias_rec = alias_gen (fun x -> Dep_conf.Alias_rec x)
```

Comment:
Wouldn't something like this be simpler (and perhaps even shorter) at the (small) expense of having to pass ~dir and s

```
let alias_rec ~dir s = Dep_conf.Alias_rec (make_alias_name ~dir s)
```
?

Reply:
Sure, I don't have a preference. I changed it to the above style.